frontend/src/employee-frontend/components/reports/PlacementCount.tsx

```diff
@@ -158,6 +158,9 @@ export default React.memo(function PlacementCount() {
         ...areaResult.daycareResults.map((daycareResult) => {
           return {
             ...daycareResult,
+            calculatedPlacements: daycareResult.calculatedPlacements
```

Comment:
I suggest using calculatedPlacements.toLocaleString('fi-FI') instead "1.23" -> "1,23"

Reply:
Switched to locale strings.